tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/media_storage.py

```diff
@@ -91,12 +62,72 @@ def confirm_select_media_location(self):
         Wait(self.marionette).until(expected.element_displayed(*self._default_change_ok_button_locator))
 
     def pick_media_location(self,location):
-        if location is 'Internal':
+        if location == 'Internal':
             _selection_locator = (By.CSS_SELECTOR, '[data-option-index="0"]')
-        elif location is 'SD Card':
+        elif location == 'SD Card':
             _selection_locator = (By.CSS_SELECTOR, '[data-option-index="1"]')
-        self.marionette.find_element(*_selection_locator).tap()
+        else:
+            raise AttributeError('{} is not a media supported in the test'.format(location))
+        element = Wait(self.marionette).until(expected.element_present(*_selection_locator))
+        Wait(self.marionette).until(expected.element_displayed(element))
+        element.tap()
         self.marionette.find_element(*self._default_change_ok_button_locator).tap()
         self.apps.switch_to_displayed_app()
         Wait(self.marionette).until(expected.element_displayed(*self._media_location_selector_locator))
 
+    class StorageRegion(PageRegion):
```

Comment:
This class is duplicated. Either we define it it, or we do it `gaiatest/apps/settings/regions/storage_region.py`. But we have to delete one of them